client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -230,7 +230,8 @@ export class BpmnEditor extends CachedComponent {
       'searchPad.closed',
       'searchPad.opened',
       'popupMenu.opened',
-      'popupMenu.closed'
+      'popupMenu.closed',
+      'elementTemplates.select'
```

Comment:
What feels a little bit magically to me is that we don't react on modal closing to update the menu. 

This is a theoretic case as we seem to have enough update checks in place to re-trigger menu update after close (refreshing available actions).